core/src/main/java/nl/nn/adapterframework/http/rest/ApiListener.java

```diff
@@ -50,12 +50,14 @@
 
 	private MediaTypes consumes = MediaTypes.ANY;
 	private MediaTypes produces = MediaTypes.ANY;
+	private ContentType producedContentType;
 	private String multipartBodyName = null;
 
 	private IReceiver receiver;
 
 	private ClassLoader configurationClassLoader = Thread.currentThread().getContextClassLoader();
 	private String messageIdHeader = AppConstants.getInstance(configurationClassLoader).getString("apiListener.messageIdHeader", "Message-Id");
+	private String charset = null; //Use system default or UTF-8?
```

Comment:
If a charset is required, UTF-8 should be used by default. For binary content no charset must be used.